packages/core/src/modules/credentials/repository/credentialMetadataTypes.ts

```diff
@@ -0,0 +1,12 @@
+export enum CredentialMetadataKeys {
+  IndyCredential = '_internal/indyCredential',
+  IndyRequest = '_internal/indyRequest',
+}
+
+export type CredentialMetadata = {
+  [CredentialMetadataKeys.IndyCredential]: {
+    schemaId?: string
+    credentialDefinitionId?: string
+  }
+  [CredentialMetadataKeys.IndyRequest]: Record<string, unknown>
```

Comment:
Would still be good to use the indy cred req type, even though it is `Record<string, unknown>`
```suggestion
  [CredentialMetadataKeys.IndyRequest]: CredReqMetadata
```